doc/source/serve/doc_code/hpu_inference_serve_deepspeed.py

```diff
@@ -0,0 +1,245 @@
+import asyncio
+import tempfile
+
+from ray.util.queue import Queue
+from ray import serve
+from ray.runtime_env import RuntimeEnv
+import ray
+import torch
+from transformers import TextStreamer
+from transformers import AutoTokenizer, AutoModelForCausalLM, AutoConfig
+from starlette.responses import StreamingResponse
+import deepspeed
+from optimum.habana.checkpoint_utils import (
+    get_ds_injection_policy,
+    write_checkpoints_json,
+)
+
+# the default port used by torch
+TORCH_DISTRIBUTED_DEFAULT_PORT = 29500
+# habana required variables
+HABANA_ENVS = {
+    "PT_HPU_LAZY_ACC_PAR_MODE": "0",
+    "PT_HPU_ENABLE_REFINE_DYNAMIC_SHAPES": "0",
+    "PT_HPU_ENABLE_WEIGHT_CPU_PERMUTE": "0",
+    "PT_HPU_ENABLE_LAZY_COLLECTIVES": "true",
+    "HABANA_VISIBLE_MODULES": "0,1,2,3,4,5,6,7",
+}
+
+# set PyTorch distributed related environmental variables
```

Comment:
The code's comments sometimes refer to torch as `PyTorch` and sometimes as `torch`. Could you ensure that only one term is used?